.github/workflows/main.yml

```diff
@@ -48,4 +48,28 @@ jobs:
         name: codecov-umbrella
         fail_ci_if_error: false
         path_to_write_report: ./coverage/codecov_report.txt
-        verbose: true
\ No newline at end of file
+        verbose: true
+
+  build_tf2:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7]
```

Comment:
Shouldn't we add python 3.8? 
https://www.tensorflow.org/install/pip

Reply:
Yes, we should. I'll add it.